packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -874,6 +879,7 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with Restoratio
     assert(debugCheckHasDirectionality(context));
     final TextEditingController controller = _effectiveController;
     final List<TextInputFormatter> formatters = widget.inputFormatters ?? <TextInputFormatter>[];
+    final TextSelectionControls textSelectionControls = widget.textSelectionControls ?? cupertinoTextSelectionControls;
```

Comment:
Would it work to set a default value for the selectionControls parameter instead of doing this here?  What's your opinion on that?

Reply:
That is a great idea. The only thing is that `cupertinoTextSelectionControls` is a final variable and not const. We could create a const constructor inside `TextSelectionControls`

Reply:
Ah you're right.  I think it's fine like this then.